src/Test/WordPress/HooksTest.php

```diff
@@ -42,9 +42,10 @@ public function setup()
         $this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
 
         $this->hooks = $this->getMockBuilder('\CF\WordPress\Hooks')
-            ->setMethods(array('setHooks'))
+            ->disableOriginalConstructor()
+            ->setMethods(array('__construct')) // This is a hack to make the tests work
```

Comment:
> we should be instantiating the real one.
The error was caused by instantiating. This was a really quick fix which avoided calling constructor but doesn't mock anything else. 

Mocking these [two](https://github.com/cloudflare/Cloudflare-WordPress/blob/master/src/WordPress/Hooks.php#L21) didn't seem straight forward instead I mocked the constructor to avoid it being called.